### DISCUSSION TITLE
[Question] C++ device side Sample code for IoT Hub C SDK?

### DISCUSSION THREAD

**Discussion by ilseokoh** [Azure/azure-iot-sdk-c#2184] [Q&A] [answered]
I'm looking for a C++ device side sample code for Azure IoT Hub C SDK. Please share a link if you have one.

**Comment by ericwolz** [comment, marked as answer]
Currently there is no C++ IoT device SDK.

**Reply by ghost** [reply]
Hi @ericwol-msft any updates regarding this topic ?

**Reply by ericwolz** [reply]
No updates for this topic. Currently there is no C++ IoT device SDK.
